django/contrib/sitemaps/templates/sitemap.xml

```diff
@@ -1,13 +1,16 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml">
```

Comment:
Can I ask you to explain this `xmlns:xhtml` change? No doubt it's correct, but it seems unrelated to the `hreflang` change — unless it's not?

Reply:
Update: this is as [advised by Google](https://support.google.com/webmasters/answer/189077)